From de17926dab1d68d266c0c4ab044c2ef1851c7091 Mon Sep 17 00:00:00 2001
From: Martin Erik Werner <martinerikwerner@gmail.com>
Date: Fri, 3 Apr 2015 20:29:02 +0200
Subject: [PATCH 1/5] appdate: Use template set-at-install name

This is to allow for installing as 'redeclipse-develop' and similar.
---
 src/install/nix/redeclipse.appdata.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/install/nix/redeclipse.appdata.xml b/src/install/nix/redeclipse.appdata.xml
index b52cd7b..d5487f7 100644
--- a/src/install/nix/redeclipse.appdata.xml
+++ b/src/install/nix/redeclipse.appdata.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <application>
-  <id type="desktop">redeclipse.desktop</id>
+  <id type="desktop">@APPNAME@.desktop</id>
   <metadata_license>CC-BY-SA-3.0+</metadata_license>
   <project_license>ZLIB and CC-BY-SA-3.0+</project_license>
   <description>
-- 
2.1.4

